source/install/desktop-app-changelog.rst

```diff
@@ -20,12 +20,99 @@ Desktop application changelog
 
 Latest Mattermost Desktop App releases:
 
-- `Release v5.0 <#id1>`_
-- `Release v4.7 <#id3>`_
-- `Release v4.6 <#id21>`_
-- `Release v4.5 <#id33>`_
-- `Release v4.4 <#id49>`_
-- `Release v4.3 <#id68>`_
+- `Release v5.1 <#id1>`_
+- `Release v5.0 <#id3>`_
+- `Release v4.7 <#id20>`_
+- `Release v4.6 <#id38>`_
+- `Release v4.5 <#id50>`_
+- `Release v4.4 <#id66>`_
+
+Release v5.1
+--------------
+
+**Release day: 2022-05-16**
+
+**Download Binaries:** `Mattermost Desktop on GitHub <https://github.com/mattermost/desktop/releases/latest>`_
+
+Compatibility
+~~~~~~~~~~~~~~~
+
+- Desktop Apps are required to be used with any `supported Extended Support Release or a newer Mattermost server version <https://docs.mattermost.com/upgrade/release-lifecycle.html>`_.
+
+Breaking Changes / Upgrade Notes
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- 
+
+Highlights
+~~~~~~~~~~~~~~~
+
+- Added a Desktop App auto-updater. The app now automatically checks for new updates on app start up.
+
+Improvements
+~~~~~~~~~~~~~~~
```

Comment:
Mac: Mattermost can now be installed on the Mac App Store (link). Even if you’re already using Mattermost desktop on Mac, you can download and install it via the Mac App Store to access future automatic updates.

Reply:
We are still providing Mac builds on GitHub, so we should find a way to communicate that while those are fine to install, they do not support auto-updates.